Makefile

```diff
@@ -0,0 +1,13 @@
+ST2USER?=st2admin
+ST2PASSWORD?=Ch@ngeMe
+
+all: setup
+
+setup:
+	mkdir -p ~/.ssh
+	cp -i ssh/id_busybee* ~/.ssh
+	chmod 600 ~/.ssh/id_busybee
+	ssh -i ~/.ssh/id_busybee root@localhost /usr/bin/setup_container.sh ${ST2USER} ${ST2PASSWORD}
```

Comment:
I've removed all the ssh related stuff from the Makefile. Now, the `setup` target currently just runs `htpasswd` and `st2 login` so the user doesn't have to manually.